3rdparty/python/BUILD

```diff
@@ -28,6 +28,12 @@ python_requirements(
     resolve="pytest",
 )
 
+python_requirements(
+    name="pbs-script",
+    source="pbs-script-requirements.txt",
+    resolve="pbs-script",
+)
```

Comment:
Should this be added to  build-support/bin/generate_builtin_lockfiles.py, or are 'script' lockfiles different from builtin ones?

Reply:
The `pbs-script` resolve is only intended for use within the Pants repository itself, and so does not seem to me like a "tool" lockfile which is bundled with Pants for use by end users. So it can probably be excluded from this script.